apps/survey/src/components/prompts/standard/MealTimePrompt.vue

```diff
@@ -1,8 +1,19 @@
 <template>
-  <card-layout v-bind="{ food, meal, prompt, section, isValid }" @action="action">
+  <card-layout
+    v-bind="{ food, meal, prompt, section, isValid }"
+    @action="action"
+  >
     <v-card-text class="pt-2 meal-time-prompt">
       <v-form @submit.prevent="action('next')">
+        <time-picker
```

Comment:
please use dynamic component instead of switch

components can be registered as 
```
components: {clock: 'v-time-picker', simple: 'time-picker'}
```
and then used as 
```
<component :is="prompt.style"  v-model="state" :allowed-minutes="allowedMinutes" :format="prompt.format">
```